### DISCUSSION TITLE
Single Sign On through auth0

### DISCUSSION THREAD

**Discussion by steveeq1** [umputun/remark42#1543] [Ideas]
Is there a way to have Single Sign On (SSO) through auth0?

**Comment by umputun** [comment]
I think this would relatively simple thing to do, but the change has to be in https://github.com/go-pkgz/auth first, and after this adding a new provider option to remark42 will be easy

_moved to discussions_

**Reply by ribtoks** [reply]
@umputun go-pkgz/auth claims to support a custom oauth2 provider. Is there a way to use it in remark42?

**Reply by paskal** [reply]
You'll need to build your custom provider into go-pkgz/auth, fork remark42 and build it on top of your own auth library version. I do not think it's possible without code changes.
